solr/contrib/ltr/example/solrconfig.xml

```diff
@@ -876,11 +875,7 @@
        <str name="indent">true</str>
        <str name="df">text</str>
      </lst>
-<arr name="last-components">
-      <!-- Use the component in your requestHandler -->
-      <str>ltrComponent</str>
-    </arr>
-  </requestHandler>
+ </requestHandler>
```

Comment:
grep for other "searchComponent" sections in code or other configs and remove them too.  I think some tests do their own thing